components/layout_2020/positioned.rs

```diff
@@ -69,14 +69,18 @@ impl AbsolutelyPositionedBox {
         display_inside: DisplayInside,
         contents: Contents,
     ) -> Self {
+        let data = PropagatedBoxTreeData::default();
+        println!("------ abso: {data:?}");
```

Comment:
Remove this, seems the cause for the timeouts.

Reply:
Whoops. Nice catch. This is gone now.